### ISSUE TITLE
Win64 builds

### ISSUE THREAD

**Issue by brian6932** [sylpheed-mail/sylpheed#14]
It's 2023, a Win64 bin build should really exist, and be prioritized over the Win32 one.

**Comment by ant-222** [comment]
As a happy user of the 32-bit version of *Sylpheed* on my old machine with *Windows XP*, may I ask you why a 64-bit binary is requied? Is there a situation where the 32-bit one does not work?

**Comment by brian6932** [comment]
Is that a joke?
1. No one said remove the 32bit build
1. `x86` is an ancient ISA, we have `x86-64`, `x86-64-v2`, `x86-64-v3`, and `x86-64-v4`, which each allow the compiler to optimize for more modern instruction sets

**Comment by ant-222** [comment]
No, it is not a joke. I shall be grateful if the 32-bit binary shall continue to be provided. On the other hand, I doubt *Sylpheed* will gain any visible performance improvement because of the more optimal machine code. It will not be checking mail faster... If, however, I am wrong and your request will solve or at least widen someknown performnce bottlenecks, then I am all for it!

**Comment by brian6932** [comment]
https://github.com/AlienCowEatCake/sylpheed-windows This repo seems to provide native Windows arm64 and x64 builds!
Also it seems like the beta version was pulled from the official site: https://sylpheed.sraoss.jp/en/download.html#development.
